src/util/config-file.js

```diff
@@ -14,6 +14,12 @@ function readConfig(configPath) {
 }
 
 function writeConfig(configPath, content) {
+  if (content.projectOwner === '') {
```

Comment:
Instead of `''`, why don't we do a simple falsy check? `if (!content.projectOwner)`

This has the added benefit of working if the value is set to `null` or `undefined` somehow.

We should do that for all the checks I think.

Reply:
@kentcdodds thank you for your comment, i however feel that it could be a big pr, would you like me to add that in the current pr?

Reply:
sure will get on it

Reply:
i didnt knew this https://stackoverflow.com/a/8693015.
Thank you @kentcdodds